docs/gallery/ontology.py

```diff
@@ -0,0 +1,130 @@
+"""
+Ontology
+=============================
+
+This is a user guide for interacting with the assorted ``Ontology`` classes, all
+of which are experimental and are subject to change in future releases.
+Please provide feedback to the HDMF team so that we can improve the structure
+and usability for your use cases.
+
+"""
+
+###############################################################################
+# Introduction
+# ------------------------------------------------------
+# With the :py:class:`~hdmf.ontology.Ontology` class and the subsequent subclasses
+# that inherit from :py:class:`~hdmf.ontology.Ontology`, users can more easily
+# link ontologies and ontology entities to
+# :py:class:`~hdmf.common.resources.ExternalResources`.
+# Fully integrated with the :py:class:`~hdmf.container.Container` class, users
+# can add ontologies through two approaches. Firstly, users can use the
+# :py:func:`~hdmf.container.Container.add_ontology` method within the
+# :py:class:`~hdmf.container.Container` class to add an ontology as a
+# :py:class:`~hdmf.common.resources.Resource and add an
+# ontology entity as an :py:class:`~hdmf.common.resources.Entity' to the
+# :py:class:`~hdmf.common.resources.ExternalResources` within the NWBFILE.
+# Lastly, users are able to link ontologies to a :py:class:`~hdmf.container.Container`
+# without immediatly adding to :py:class:`~hdmf.common.resources.ExternalResources`.
+# A typical use case of the latter would be to create a controlled set of values
+# within data. The user is able to add this "linked" ontology to
+# :py:class:`~hdmf.common.resources.ExternalResources` via (TBD on function).
+
+###############################################################################
+# Creating a LocalOntology
+# ------------------------------------------------------
+# Users are able to create their own ontologies that will be stored locally on
+# their machine with :py:class:`~hdmf.ontology.LocalOntology`. Users will define
+# the entities of the ontology within a python dictionary. To add new entries,
+# it is required that the entries have both the entity id and the corresponding uri.
+# Once added, users can add or remove entries with built-in methods.
+# :py:class:`~hdmf.ontology.LocalOntology` also supports the ability to write
+# and read the ontology to and from a YAML file.
```

Comment:
Can you make this in second person? Instead of "Users are able to create their own ontologies that will be stored locally...", I would prefer "Create your own ontologies that will be stored locally..."